### DISCUSSION TITLE
O site Portugol Web Studio está com instabilidade encerra quando erra alguma coisa no terminal

### DISCUSSION THREAD

**Discussion by ccarreiro** [dgadelha/Portugol-Webstudio#203] [Ideias e sugestões]
![Screenshot_20240415-193658](https://github.com/dgadelha/Portugol-Webstudio/assets/34811379/1088dec3-063c-49bb-a4ed-27f78a272eaf)

**Comment by dgadelha** [comment]
Corrigido em https://github.com/dgadelha/Portugol-Webstudio/commit/8672f4287d611e07e9c92235da1bf8be721979b0

Obrigado pelo report!
